packages/editor/src/AppModel/AppModel.ts

```diff
@@ -12,6 +13,7 @@ import {
 import { genComponent } from './utils';
 
 export class AppModel implements IAppModel {
+  widgets: Record<string, Widget> = {};
```

Comment:
`AppModel` is the model of application schema. Widgets are not part of application schema, so they should not be registered here.

Reply:
OK, I will use a new model to manage the widgets.